src/test/kotlin/uk/gov/justice/hmpps/offendersearch/controllers/OffenderSearchNomsListAPIIntegrationTest.kt

```diff
@@ -0,0 +1,134 @@
+package uk.gov.justice.hmpps.offendersearch.controllers
+
+import io.restassured.RestAssured
+import org.assertj.core.api.Assertions
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.springframework.http.MediaType
+import uk.gov.justice.hmpps.offendersearch.dto.OffenderDetail
+
+class OffenderSearchNomsListAPIIntegrationTest : LocalstackIntegrationBase() {
+
+    @BeforeEach
+    fun setUp() {
+        loadOffenders(
+                OffenderReplacement(nomsNumber = "G8020GG"),
+                OffenderReplacement(nomsNumber = "G8020GH"),
+                OffenderReplacement(nomsNumber = "G8020GI", deleted = true),
+        )
+    }
+
+    @Test
+    fun `not allowed to do a search without COMMUNITY role`() {
+        RestAssured.given()
+                .auth()
+                .oauth2(jwtAuthenticationHelper.createJwt("ROLE_BINGO"))
+                .contentType(MediaType.APPLICATION_JSON_VALUE)
+                .body("[\"X00001\",\"X00002\"]")
```

Comment:
This is fine but a nice tip for the future - you don't need to escape special chats with Kotlin's triple quote, e.g.
`"""["X00001","X00002"]"""`